### PR TITLE
Fix empty tooltip and missing required tooltip

### LINKED ISSUE #757: Tooltip is empty
I'm seeing that "Example 17" on the Fuller Demo shows a tooltip that doesn't have any content.

Screen Shot 2019-06-18 at 12 16 55 PM

### PR DESCRIPTION
**Pull request by jgreben**
Fixes #757 and #392

### CODE REVIEW

File: src/components/editor/property/PropertyLabel.jsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export class PropertyLabel extends Component {
       <span key={key}>{property.propertyLabel}</span>
     )
 
-    url !== undefined ? title.push(urlLabel) : property.remark !== undefined ? title.push(toolTipLabel) : title.push(plainLabel)
+    url !== undefined ? title.push(urlLabel) : property.remark !== undefined && property.remark !== '' ? title.push(toolTipLabel) : title.push(plainLabel)
```
Comment (jcoyne) [inline src/components/editor/property/PropertyLabel.jsx]: can you use ` _.isEmpty(property.remark)`?

**Comment by jgreben** [comment]
> Screen Shot 2019-06-19 at 8 48 04 AM
> 
> This doesn't seem easy to read. I don't think we want both a title and a popover menu. Maybe @astridu can help?

I made this change pending @astridu's feedback:
Screen Shot 2019-06-19 at 11 51 29 AM

**Comment by astridu** [comment]
Yes, the second example looks much better (just the 1 tooltip). Thank you!

**Comment by jcoyne** [comment]
I'm seeing this now.  Since we have the tooltip, should we get rid of the `<abbr/>`?

Screen Shot 2019-06-20 at 9 24 45 AM

**Comment by jcoyne** [comment]
@michelleif @jermnelson what sort of commitments to we have to accessibility?  I'm pretty sure the `<abbr>` with a title are useful for screen readers.  I'm not sure the popover is useful for anyone not using a mouse.

**Comment by ndushay** [comment]
would it be stupid to add a css styling that doesn't do the popover (because we display it another way?)

**Comment by jermnelson** [comment]
> @michelleif @jermnelson what sort of commitments to we have to accessibility? I'm pretty sure the `<abbr>` with a title are useful for screen readers. I'm not sure the popover is useful for anyone not using a mouse.

I would like us to be more accessible. We're running this elint plugin: https://www.npmjs.com/package/eslint-plugin-jsx-a11y with some rule exceptions.

**Comment by jcoyne** [comment]
So do we know if a screenreader can focus on a `<sup>` element that has a single child marked `aria-hidden="true"`?

**Comment by jgreben** [comment]
I added a tabIndex to the `sup` to make it keyboard accessible, and role-tooltip for ARIA. Do you  think this will address the accessibility issues?

**Comment by jcoyne** [comment]
Yes, that probably ought to do it.

**Pull request merged**